src/openrct2/peep/staff.c

```diff
@@ -469,13 +468,13 @@ void game_command_set_staff_patrol(sint32 *eax, sint32 *ebx, sint32 *ecx, sint32
             return;
         }
         rct_peep *peep = &sprite->peep;
-        sint32 patrolOffset = peep->staff_id * (64 * 64 / 8);
```

Comment:
64 * 64 / 8 = 512, not 128.

Reply:
Everything is divided by 4 again, because we're indexing into uint32s, but the pointer arithmetic works in bytes (same for the other cases).